pkg/scaffold/scaffold.go

```diff
@@ -185,14 +166,19 @@ func (s *Scaffold) Execute(u *model.Universe, options input.Options, files ...in
 		}
 	}
 
-	if u.Boilerplate == "" {
-		u.Boilerplate = s.Boilerplate
-	}
-
 	if err := s.defaultOptions(&options); err != nil {
 		return err
 	}
 
+	// Set default values for the universe
+	if u.Project == nil {
+		u.Project = &s.Project
```

Comment:
` cannot use &s.Project (type **input.ProjectFile) as type *input.ProjectFile in assignment`

Reply:
Yes, the `&` needs to be removed. Saw that in Travis too